spirv_hlsl.cpp

```diff
@@ -3209,6 +3633,13 @@ void CompilerHLSL::emit_legacy_uniform(const SPIRVariable &var)
 
 void CompilerHLSL::emit_uniform(const SPIRVariable &var)
 {
+	// make rgen payload static
```

Comment:
I think ray payloads could have it's own emit function, this feels a little clumsy.

Reply:
Done